app/components/Sidebar/components/DocumentLink.tsx

```diff
@@ -275,7 +276,7 @@ function DocumentLink(
             </DropToImport>
           </div>
         </Draggable>
-        {manualSort && (
+        {manualSort && isDraggingAnyDocument && (
```

Comment:
Don't render the cursor unless we're drag-n-dropping'

Note: We were already doing this for collections!